engine/apps/telegram/updates/update_handlers/button_press.py

```diff
@@ -45,6 +45,14 @@ def process_update(self) -> None:
             self.update.callback_query.answer(PERMISSION_DENIED, show_alert=True)
             logger.info(f"User {user} has no permission to trigger '{fn.__name__}'")
 
+    def _get_alert_group_from_message(self) -> Optional[AlertGroup]:
+        alert_group = None
+        message = self.update.message
+        if message:
```

Comment:
_nit_ ([walrus operator](https://realpython.com/python-walrus-operator/) can be a bit less verbose sometimes 🤓 )
```suggestion
        if message := self.update.message:
```